### PR TITLE
Load language files from mods

### PR DESCRIPTION
**Pull request by BoogieMonster1O1**
fixes another bug with commands

### CODE REVIEW

File: fabric-resource-loader-v0/build.gradle [hunk 1]
```diff
@@ -1,2 +1,2 @@
 archivesBaseName = "fabric-resource-loader-v0"
-version = getSubprojectVersion(project, "0.2.2")
+version = getSubprojectVersion(project, "0.3.0")
```
Comment (hYdos) [inline fabric-resource-loader-v0/build.gradle:2]: this is just a bug fix. a major increment is not needed

**Review by hYdos — CHANGES_REQUESTED** [review]

File: fabric-resource-loader-v0/build.gradle (hunk 1 above)
Comment (BoogieMonster1O1) [reply fabric-resource-loader-v0/build.gradle:2]: That isn’t a major increment
and it’s adding new features without breaking existing ones
hence minor increment

Comment (hYdos) [reply fabric-resource-loader-v0/build.gradle:2]: a bug fix is the 3rd number not the second.

Comment (BoogieMonster1O1) [reply fabric-resource-loader-v0/build.gradle:2]: doshy please read what i've written
> and it’s adding new features without breaking existing ones
hence minor increment

while this fixes a bug that's not its main purpose

**Review by hYdos — APPROVED** [review]

**Pull request merged**